src/components/modals/EditDialogAdmin.tsx

```diff
@@ -398,9 +397,7 @@ class EditDialogAdmin extends React.Component<Props, ComponentState> {
                         onAddTag={this.props.onAddTag}
                         onRemoveTag={this.props.onRemoveTag}
                     />
-                    <TrainingStatusContainer
-                        app={this.props.app}
-                    />
+                    <b></b>
```

Comment:
Note: Use of `<b>` tag here is convention I've seen to indicate it's just a placeholder for the grid.  Similar to `div` it has no effect on the page.